### PR TITLE
feat(rtp): RTCP parsing

### PR DESCRIPTION
**Pull request by Tigge**
Extend media stream library with basic RTCP parsing. Supports all things
from RTP: A Transport Protocol for Real-Time Applications
(RFC 3550).

Unknown package types will be parsed as generic RTCP messages which can
be further parsed by the application if need.

Includes a small test suite for RTCP message parsing.

### CODE REVIEW

File: examples/browser/camera/simple-player.js [hunk 1]
```diff
@@ -44,10 +44,21 @@ const play = (host, encoding = 'h264') => {
...
+  }
+
   pipeline.ready.then(() => {
     pipeline.rtsp.play()
   })
 
+  mediaElement.addEventListener('ended', (e) => {
+    // setTimeout(() => play(host, encoding), 0)
+  })
+
```
Comment (steabert) [inline examples/browser/camera/simple-player.js:61]: remove

File: examples/browser/camera/simple-player.js [hunk 2]
```diff
@@ -44,10 +44,21 @@ const play = (host, encoding = 'h264') => {
     rtsp: { uri: `rtsp://${host}/axis-media/media.amp?videocodec=${encoding}` },
     mediaElement,
   })
+
+  pipeline.rtsp.onRtcp = (rtcp) => {
```
Comment (steabert) [inline examples/browser/camera/simple-player.js:48]: Short comment what this does.

**Review by steabert — APPROVED** [review]
Nice work.

File: examples/browser/camera/simple-player.js (hunk 2 above)
Comment (Tigge) [reply examples/browser/camera/simple-player.js:49]: Done

File: examples/browser/camera/simple-player.js (hunk 1 above)
Comment (Tigge) [reply examples/browser/camera/simple-player.js]: Done

**Pull request merged**